File: project_flutter/src/main/java/com/example/project_flutter/entity/Software.java
package com.example.project_flutter.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;

/**
 * @author 西云研究所
 * @date 2021/10/18
 */
@Component
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Software {
    private long software_id;
    private long catogory_id;
    private String software_name;
    private String software_need;
}

File: untitled1/lib/project/entity/Software.dart
class Software {
  int software_id;
  int catogory_id;
  String software_name;
  String software_need;

  Software(
      {this.catogory_id,
      this.software_id,
      this.software_name,
      this.software_need});

  factory Software.fromJson (Map<String, dynamic> parsedJson){
    return Software(
      catogory_id: parsedJson["catogory_id"],
      software_id: parsedJson["software_id"],
      software_name : parsedJson["software_name"],
      software_need: parsedJson["software_need"]
    );
  }

}
